pubnub-core/src/lib.rs

```diff
@@ -9,6 +9,10 @@
 //!
 //! Build your own client, or use preconfigured [`pubnub-hyper`](pubnub-hyper).
 
```

Comment:
I'm not sure I fully understand the comments here. What changed in 1.59? Is it just the location of the lint rule that changed?

Reply:
The "broken_intra_doc_links" rule's name changed to "rustdoc::broken_intra_doc_links. The older build doesn't recognize the new name, and vice versa. We have the linter rules strict enough that this, which would normally be a warning, breaks the build. The directives here relax that rule, and we can tighten it back up when we eventually require a minimum version that is 1.59.0 or later.

Reply:
fair enough